go.mod

```diff
@@ -53,17 +53,18 @@ require (
 	gopkg.in/fsnotify.v1 v1.4.7
 	gopkg.in/yaml.v2 v2.4.0
 	helm.sh/helm/v3 v3.7.0
-	k8s.io/api v0.23.0
-	k8s.io/apimachinery v0.23.0
+	k8s.io/api v0.23.6
```

Comment:
Wow. 1.23.6 released :smile: I'd rather set this to k0s's "official" k8s version of 1.23.5, and upgrade these versions in a separate PR that bumps k8s to 1.23.6.

Reply:
Yeah, removed it